frontend/src/features/admin-form/settings/components/FormLimitToggle.tsx

```diff
@@ -90,7 +90,7 @@ const FormLimitBlock = ({
         ref={inputRef}
         // min={currentResponseCount + 1}
```

Comment:
remove unused prop (sorry i know not you LOL)

Reply:
<img width="582" alt="image" src="https://user-images.githubusercontent.com/25571626/195248364-a3333783-5303-4ba5-872a-69a583879e50.png">

[3e0867d](https://github.com/opengovsg/FormSG/pull/5115/commits/3e0867d55278e179136701f7e5404b60d64f692d)